runtime/src/lib.rs

```diff
@@ -300,16 +354,18 @@ impl pallet_balances::Config for Runtime {
     type ReserveIdentifier = [u8; 8];
 }
 
+parameter_types! {
+	/// Relay Chain `TransactionByteFee` / 10
+	pub const TransactionByteFee: Balance = 10 * MICROUNIT;
```

Comment:
in 'master' we're using `10 * MILLICENTS`, is this new value equivalent?

Reply:
Its similar to master now